vector/src/main/java/im/vector/preference/VectorPreference.kt

```diff
@@ -83,6 +99,35 @@ open class VectorPreference : Preference {
                 summary.setTypeface(null, mTypeface)
             }
 
+            //cancel existing animation (find a way to resume if happens during anim?)
+            currentHighlightAnimator?.cancel()
+            if (isHighlighted) {
+                val colorFrom = 0
```

Comment:
Should use a constant type `Color.TRANSPARENT` rather than magic number 0 :)

Reply:
ok